pegen/pegen.c

```diff
@@ -1295,3 +1301,350 @@ seq_delete_starred_exprs(Parser *p, asdl_seq *kwargs)
     }
     return new_seq;
 }
+
+
+//// STRING HANDLING FUNCTIONS ////
```

Comment:
Might note that these are all copied from Python/ast.c with slight modifications to compensate for the use of `Parser *p` instead of `struct compiling *c`. (And what else? I noticed some functions lacking a `node *n` argument as well.)